Packs/Nexthink/Integrations/Nexthink/Nexthink.yml

```diff
@@ -0,0 +1,64 @@
+category: Utilities
+commonfields:
+  id: Nexthink
+  version: -1
+configuration:
+- defaultvalue: connector.nexthink.com
+  display: Engine Host (e.g. connector.nexthink.com)
+  name: url
+  required: true
+  type: 0
+- defaultvalue: "1671"
+  display: Nexthink Connector Port
+  name: port
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Nexthink helps IT teams deliver on the promise of the modern digital
+  workplace. Nexthink is the only solution to provide enterprises with a way to visualize,
+  act and engage across the entire IT ecosystem to lower IT cost and improve digital
+  employee experience.
+display: Nexthink
+name: Nexthink
+script:
+  commands:
+  - arguments:
+    - description: Endpoint IP Address
+      name: ipaddress
+    - description: Endpoint Hostname
+      name: hostname
+    description: Verify antivirus/antispyware status
+    name: nt_compliance_check
+  - arguments:
+    - description: Endpoint Hostname
+      name: hostname
```

Comment:
Please note that you are missing the `required` and the `default` fields, let's add them.